tests/ansible_vagrant/README.md

```diff
@@ -0,0 +1,28 @@
+# Ansible playbook
+An ansible playbook to install a simple k8s cluster with a master node and two worker nodes. The CNI plugin is pcn-k8s. Vagrant is used to provide the VMs.
+
+## Requirements
+ * Vagrant 2.2.5 or higher
+ * Ansible 2.8.5 or higher
+
+## Usage
+ * ansible-playbook master-playbook.yml
```

Comment:
This should be shown as code by using back-ticks:
 `ansible-playbook master-playbook.yml`